$MirOS: ports/misc/mc/patches/patch-doc_mcview_1_in,v 1.1.7.1 2005/03/18 15:50:27 tg Exp $
--- doc/mcview.1.in.orig	Wed Jun  8 12:26:45 2005
+++ doc/mcview.1.in	Mon Feb 20 18:32:46 2006
@@ -68,14 +68,14 @@ The help file for the program.
 .I @prefix@/share/mc/mc.ini
 .IP
 The default system-wide setup for GNU Midnight Commander, used only if
-the user's own ~/.mc/ini file is missing.
+the user's own ~/.etc/mc/ini file is missing.
 .PP
 .I @prefix@/share/mc/mc.lib
 .IP
 Global settings for the Midnight Commander.  Settings in this file
-affect all users, whether they have ~/.mc/ini or not.
+affect all users, whether they have ~/.etc/mc/ini or not.
 .PP
-.I $HOME/.mc/ini
+.I $HOME/.etc/mc/ini
 .IP
 User's own setup.  If this file is present, the setup is loaded from
 here instead of the system-wide startup file.
